concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -2,22 +2,49 @@
 set -eu
 
 SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
-
 export TARGET_CONCOURSE=deployer
 # shellcheck disable=SC2091
 $("${SCRIPT_DIR}/environment.sh" "$@")
-"${SCRIPT_DIR}/fly_sync_and_login.sh"
 
-env="${DEPLOY_ENV}"
-pipelines_to_update="${PIPELINES_TO_UPDATE:-create-bosh-cloudfoundry destroy-cloudfoundry destroy-microbosh autodelete-cloudfoundry failure-testing}"
+download_git_id_rsa() {
+  git_id_rsa_file=$(mktemp)
+
+  aws s3 cp "s3://${env}-state/git_id_rsa" "${git_id_rsa_file}"
```

Comment:
Shall I? 

That works in the latests versions of AWS cli, but not some previous. For instance, the awscli distributed in Ubuntu stable does not support it.

It might cause some trouble to somebody not using the AWS client specified in the `requirements.txt`.